homeassistant/components/velbus/config_flow.py

```diff
@@ -82,3 +85,40 @@ async def async_step_user(
             ),
             errors=self._errors,
         )
+
+    async def async_step_usb(self, discovery_info: usb.UsbServiceInfo) -> FlowResult:
+        """Handle USB Discovery."""
+        vid = discovery_info.vid
+        pid = discovery_info.pid
+        serial_number = discovery_info.serial_number
+        device = discovery_info.device
+        manufacturer = discovery_info.manufacturer
+        description = discovery_info.description
```

Comment:
If you are only interested in the `device`, then I don't think you need to create 6 temporary variables.